### DISCUSSION TITLE
Constant access to the mic

### DISCUSSION THREAD

**Discussion by IvanKatsiia** [kyleneideck/BackgroundMusic#759] [General]
Just curious: why does this app need constant access to the mic?

**Comment by gchilds** [comment]
The app **doesn't** need constant access to the mic, but to work it does need access to its own virtual audio device, which macOS considers a microphone because of the device's input stream / channel. Kyle (and other virtual audio device and app authors) could arrange direct access to their own virtual devices without using a CoreAudio input stream, but it's a lot of work just to get rid of an annoying orange dot and mic access request.

**Reply by IvanKatsiia** [reply]
got it, thank you!
